src/Dropdown.php

```diff
@@ -2990,7 +2990,11 @@ public static function getDropdownValue($post, $json = true)
                         // Ensure condition contains table name to prevent ambiguity with fields from `glpi_entities` table
                         $where["$table.$key"] = $value;
                     } else {
-                        $where[$key] = $value;
+                        if ($key === 'NOT' && isset($where['NOT'])) {
+                            $where[] = ['NOT' => $value];
+                        } else {
+                            $where[$key] = $value;
+                        }
```

Comment:
Maybe a better way would be to do that:
```suggestion
                        if (isset($where[$key])) {
                            $where[] = [$key => $value];
                        } else {
                            $where[$key] = $value;
                        }
```